### PR TITLE
feat: Add working directory input

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: .github/workflows/deploy-appengine-it.yml [hunk 1]
```diff
@@ -143,23 +140,27 @@ jobs:
...
+      continue-on-error: true
+      with:
+        project_id: ${{ secrets.APPENGINE_DEPLOY_PROJECT_ID }}
+        deliverables: this/path/doesnt/exist/app.yaml
+        promote: false
+    - name: Catch failure
+      run: |-
+        if [ ${{ steps.deliverable.outcome }} != 'failure' ]; then
+          exit 1
+        fi
```
Comment (bharathkkb) [inline .github/workflows/deploy-appengine-it.yml:166]: nit: newline

File: src/main.ts [hunk 2]
```diff
@@ -26,12 +27,41 @@ async function run(): Promise<void> {
...
+    // Validate deliverables
+    const allDeliverables = deliverables.split(' ');
+    if (allDeliverables[0] == '') {
+      allDeliverables[0] = 'app.yaml';
+    }
+    let notFound = false;
+    for (const deliverable of allDeliverables) {
+      if (!fs.existsSync(deliverable)) {
+        core.error(deliverable + ' is not in path.');
+        notFound = true;
```
Comment (bharathkkb) [inline src/main.ts:54]: Should we fail fast if one of the deliverable is notFound?

File: src/main.ts [hunk 3]
```diff
@@ -26,12 +27,41 @@ async function run(): Promise<void> {
...
+      }
+    }
+    if (notFound) {
+      const message =
+        'Deliverables can not be found. ' +
+        'Check `working_directory` and `deliverables` input paths.';
+      // core.setFailed();
```
Comment (bharathkkb) [inline src/main.ts:61]: nit: remove commented code

File: src/main.ts [hunk 4]
```diff
@@ -26,12 +27,41 @@ async function run(): Promise<void> {
...
     const version = core.getInput('version');
     const promote = core.getInput('promote');
     const serviceAccountKey = core.getInput('credentials');
 
+    // Change working directory
+    if (cwd) {
+      try {
+        process.chdir(cwd.trim());
+      } catch (error) {
+        core.setFailed(error.message);
```
Comment (bharathkkb) [inline src/main.ts:42]: Shouldn't we just throw an error with the correct message and let the parent try/catch set to failed?

File: .github/workflows/deploy-appengine-it.yml (hunk 1 above)
Comment (averikitsch) [reply .github/workflows/deploy-appengine-it.yml:166]: done

File: src/main.ts (hunk 4 above)
Comment (averikitsch) [reply src/main.ts:42]: Thanks for catching

File: src/main.ts (hunk 3 above)
Comment (averikitsch) [reply src/main.ts:61]: done

File: src/main.ts (hunk 2 above)
Comment (averikitsch) [reply src/main.ts:54]: sgtm

**Review by bharathkkb — APPROVED** [review]

**Comment by averikitsch** [comment]
Fixes issue #19

**Pull request merged**